air-gradient-open-air.yaml

```diff
@@ -0,0 +1,311 @@
+# Airgradient Open Air Outdoor v1.1 presoldered
+# 
+# @llamagecko, @Hendrik and @spectrumjade from AirGradient forum
+# did most of the work for this integration, see the following thread:
+# https://forum.airgradient.com/t/outdoor-monitor-esphome-configuration/823
+#
+
+substitutions:
+  id: "1"
+  devicename: "airgradient-open-air"
+  upper_devicename: "AirGradient Open Air"
+
+esphome:
+  name: "${devicename}-${id}"
+  friendly_name: "${upper_devicename} ${id}"
+  # Automatically add the mac address to the name
+  # so you can use a single firmware for all devices
+  # name_add_mac_suffix: true
+  platform: esp32
+  board: esp32-c3-devkitm-1
+
+# Enable logging
+logger:
+  baud_rate: 0
+
+# Enable Home Assistant API
+api:
+
+ota:
+  password: !secret ota_password
+
+wifi:
+  networks:
+  - ssid: !secret wifi_ssid
+    password: !secret wifi_password
+  reboot_timeout: 15min
+
+  # Enable fallback hotspot (captive portal) in case wifi connection fails
+  ap:
+    ssid: "${upper_devicename} Hotspot"
+    password: !secret fallback_ssid_password
+
+# Used to support POST request to send data to AirGradient
+# https://esphome.io/components/http_request.html
+http_request:
+
+web_server:
+  port: 80
+  include_internal: true
+
+# captive_portal:
```

Comment:
I had some issues getting my wifi connected and it took me awhile to realize this was disabled. The hotspot fallback SSID doesn't really work if `captive_portal` is  disabled like it is here.